components/events/single-events/WhatIsThisEvent.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <section>
+    <h2 class="copy__title" v-text="title" />
+    <AppMosaic :mosaic-elements="mosaicElements" />
+  </section>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component, Prop } from 'vue-property-decorator'
+import { MosaicElement } from '~/components/ui/AppMosaic.vue'
+
+@Component
+export default class WhatIsThisEvent extends Vue {
```

Comment:
I'm curious why we don't use `EventTitle` or something of that nature?

Reply:
Sure, it's because this component isn't only the title but also the mosaic.

So, everything inside the red square:

<img width="1112" alt="104320063-1a913c80-54e2-11eb-916c-ba228cfd339e" src="https://user-images.githubusercontent.com/22047320/104351010-1de9ef80-5505-11eb-90c5-9e44c5c17d32.png">